frontend/src/metabase-types/api/table.ts

```diff
@@ -81,5 +81,5 @@ export interface ForeignKey {
   origin_id: FieldId;
   destination?: Field;
   destination_id: FieldId;
-  relationship: string; // enum?
+  relationship: "Mt1";
```

Comment:
We seem to only hardcode this, there is a single place that specifies this property.
https://github.com/metabase/metabase/blob/1a3decaadadc8d2d19699857e570bbab94b362cf/src/metabase/api/table.clj#L461